### ISSUE TITLE
Geolite Integration does not work

### ISSUE THREAD

**Issue by Tobix99** [CyferShepard/Jellystat#227]
**Describe the bug**
I can't get the IPInfo Modal to open. When I'm playing the Media, I see the IP in the activity card and in the activity table, but there should be a Link to click to open the IPInfo Modal to get GeoData about the remote IP.

I've added the two Environment Variables and also got the utils/geolocateIp Endpoint to work via URL. So I think Jellystats is correctly configured.

```bash
export token="eyJc..."
export ipAddress="IP.to.check.out"

curl -X POST \
     -H "Authorization: Bearer $token" \
     -H "Content-Type: application/json" \
     -d "{\"ipAddress\": \"$ipAddress\"}" \
     https://[HOST]/utils/geolocateIp
```

**Environment Details (please complete the following information):**
 - OS: Ubuntu/Docker (latest and unstable Tag) / MacOS as Client
 - Browser: Chrome (Arc)
 - Jellystat Version: 1.1.0 and unstable
 - Jellyfin Version: 10.9.8

**To Reproduce**
1. Play a Video
2. Try to click on the IP (should open the Modal as seen here: [GitHub Link)](https://github.com/CyferShepard/Jellystat/blob/e691980b6c6b2f339fae644abc549bce77d694c0/src/pages/components/sessions/session-card.jsx#L105)

**Expected behavior**
Modal should open with infos about remote IP

**Container Logs**
```log
> jfstat@1.1.1 start
> cd backend && node server.js

[JELLYSTAT] Database exists. Skipping creation
FS-related option specified for migration configuration. This resets migrationSource to default FsMigrations
FS-related option specified for migration configuration. This resets migrationSource to default FsMigrations
[JELLYSTAT] Server listening on http://127.0.0.1:3000
```

Docker Compose File

```yaml
  jellystat:
    image: cyfershepard/jellystat:unstable
    container_name: jellystat
    restart: unless-stopped
    networks:
      - unifi
    environment:
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: mypassword
      POSTGRES_IP: jellystat-db
      POSTGRES_PORT: 5432
      JWT_SECRET: "my-secret-jwt-key"
      TZ: Europe/Berlin # timezone (ex: Europe/Paris)
      JS_GEOLITE_ACCOUNT_ID: "1..."
      JS_GEOLITE_LICENSE_KEY: "B..._..._..."
    volumes:
      - ./dockerdata/jellystat/jellystat-backup-data:/app/backend/backup-data
    depends_on:
      - jellystat-db

# and DB and reverse Proxy
```

**Comment by CyferShepard** [comment]
hey @Tobix99 , this should be fixed on unstable, can you confirm if it works?

**Comment by Tobix99** [comment]
Currently on Vacation, will do when I‘m back on Monday

**Comment by Tobix99** [comment]
@CyferShepard mhhh yes it works for IPv4 Addresses, but not for IPv6 Addresses 🤔

**Comment by CyferShepard** [comment]
Closing as Geolite bug was fixed, ipv6 is a separate issue tracked on #238
